Templates/TimerTrigger/function.json

```diff
@@ -2,7 +2,7 @@
   "disabled": false,
   "bindings": [
     {
-      "name": "myTimer",
+      "name": "timerinfo",
```

Comment:
I've updated this in https://github.com/christopheranderson/azure-webjobs-sdk-templates/commit/b732e497d2fe624ee2dfe0c96b3f587c9f5ff03b